### DISCUSSION TITLE
Task list and Check list

### DISCUSSION THREAD

**Discussion by rossanoparis** [community/community#62440] [New to GitHub] [answered]
Hi there, I'd like to ask a qustion regarding the use of "Task list" & "Check List".

In a issue if I insert the following code I get what I call a "Check List".
It is useful to inform about some options which are present in a issue.
Such kind of "Check List" doesn't own any ineraction with the collaborators, its pourpose is related to offer some information about the issue itself.
```
<ul>
 <li>[ ] Activity 1</li>
 <li>[ ] Activity 2</li>
 <li>[ ] Activity 3</li>
</ul>
```

Instead of a "Check List" I can use a "Task List", which is useful to interact with the collaborators in order to get information about the percentage of activities have been done regarding a paricular issue.
In this case the 100% represents 3 task completed.
```
- [ ] Task 1
- [ ] Task 2
- [ ] Task 3
```

But if I mix the two codes in the same issue, as shown below, I get something unexpected.
The total amount of task is 6 instead of 3.
How could I mix the two codes in order to get only 3 tasks instead of 6?

```
## Check list
<ul>
 <li>[ ] Activity 1</li>
 <li>[ ] Activity 2</li>
 <li>[ ] Activity 3</li>
</ul>

## Task list
- [ ] Task 1
- [ ] Task 2
- [ ] Task 3
```

**Comment by queenofcorgis** [comment, marked as answer]
Hey there @hobbitdev 👋🏼 

The way you've set up tasklists might be the reason it is showing six instead of three. Are you able to update your markdown to this (one code block but due to formatting it's been split in two - image should clarify below) 

```
## Check list
<ul>
 <li>[ ] Activity 1</li>
 <li>[ ] Activity 2</li>
 <li>[ ] Activity 3</li>
</ul>
```
```
```[tasklist]
## Task list
- [ ] Task 1
- [ ] Task 2
- [ ] Task 3
```
Then put three ticks under the last task aka these ``` 

It would look like this for reference: 
<img width="259" alt="Screenshot 2023-08-04 at 11 47 51 AM" src="https://github.com/community/community/assets/92824518/624fc4c6-346a-4d71-93c6-8422670a04a9">

For further reading or information, [this doc](https://docs.github.com/en/issues/managing-your-tasks-with-tasklists/creating-a-tasklist#creating-tasklists-with-markdown) may be of use as well as it has the syntax for tasklists ✔️ .

**Comment by rossanoparis** [comment]
Thank you @queenofcorgis for your reply.
Excuse me, but I don't understand your suggestion and the documentation neither.

The code below dosen't work as expected or suggested ...
![image](https://github.com/community/community/assets/33904565/34cb1d9d-7a49-4df1-ac56-1149a44593ba)

This is the result ...
As you can see the "Task list" is not shown as expected, it is shown as a code section
![image](https://github.com/community/community/assets/33904565/4fd005aa-3f50-4815-a4fe-420c7d87630e)

Am I missing something ?
Is there the following advice regarding the documentation [ [doc](https://docs.github.com/en/issues/managing-your-tasks-with-tasklists/creating-a-tasklist#creating-tasklists-with-markdown) ] What does it means
![image](https://github.com/community/community/assets/33904565/a3129b98-9a8d-4fe2-9a74-9a1d6e245f11)

**Reply by queenofcorgis** [reply]
@hobbitdev is your organization part of the private beta for task lists? That may be why you're not getting expected results with the code if you're not part of that.

**Comment by rossanoparis** [comment]
No, we are not part of it ... how could I set the organizaton to be part of it
Could you suggest a link to achive it?

**Reply by queenofcorgis** [reply]
Admission to the private beta for tasklists is limited and only available for organizations. You must be an admin to enroll your organization in the waitlist and signing up does not guarantee access. [Sign up for the waitlist here](https://github.com/features/issues/signup).

**Comment by rossanoparis** [comment]
OK, Thank you very much, I will set it on the organization I belong to.

**Reply by queenofcorgis** [reply]
Thanks for the back and forth - appreciate you signing up!
